lib/schemas/global/apiHearing.json

```diff
@@ -179,5 +179,12 @@
     "courtCentre",
     "type"
   ],
-  "additionalProperties": false
+  "additionalProperties": {

```

Comment:
Including these two as additional properties so that we don't lose the value of other schema matching in our tests if commented out. 
These additional properties will be overwritten when we are given the updated schema files by HMCTS that should accompany the multiday hearing changes